app/assets/stylesheets/components/_pagination.scss

```diff
@@ -0,0 +1,38 @@
+.pagination a, .pagination span.current, .pagination span.gap {
+  background-color: $pale-lily;
+  border: 1px solid $light-grey;
+  border-left-width: 0;
+  float: left;
+  line-height: 30px;
+  padding: 0 14px;
+  text-decoration: none;
+}
+
+.pagination {
+  border-left: 1px solid $light-grey;
+  .first{
+    border: none;
+    float: none;
+    padding: 0;
+  }
+  .prev {
+    border: none;
+    float: none;
+    padding: 0;
+  }
+  .page{
+    border: none;
+    float: none;
+    padding: 0;
+  }
+  .next{
+    border: none;
+    float: none;
+    padding: 0;
+  }
+  .last{
+    border: none;
+    float: none;
+    padding: 0;
```

Comment:
Lines 13-37 can be written as so:
```
.first, .prev, .next, .last {
  border: none;
  float: none;
  padding: 0;
} 
```

Reply:
Does `.current` need to be styled differently?

Reply:
Right, I added a darker colour for current page :)
Done in sha `d2c2b8b`